platform/ethereum/collection.go

```diff
@@ -56,7 +58,7 @@ func NormalizeCollectiblePage(collectibles []Collectible, coinIndex uint) (page
 	return page
 }
 
-func NormalizeCollectible(a Collectible, coinIndex uint) blockatlas.Collectible {
+func NormalizeCollectible(a collection.Collectible, coinIndex uint) blockatlas.Collectible {
```

Comment:
if collectible then `c`, not sure why `a`

Reply:
those are legacy code, will address them in a separate PR